installer/helm/chart/volcano/templates/controllers.yaml

```diff
@@ -150,4 +151,8 @@ spec:
               - -v=4
               - 2>&1
             imagePullPolicy: {{ .Values.basic.image_pull_policy }}
+            {{- if $controller_container_sc }}
+            securityContext:
+              {{- toYaml $controller_container_sc | nindent 14 }}
```

Comment:
Why nindent is 14 here but other places are 12?

Reply:
I followed the setting from `resources` field of the container, to be consistent. In other deployments, `resources` has nindent 12 as well, just in this one 14

Reply:
But nindent of all deployments should be 12 or 14, we should check that by `helm template` to get a correct value, right?

Reply:
I tested with `helm template` and resulting YAMLs are produced correctly and can be applied. For functionality, it doesn't matter if indents are the same across the deployments nor it matters if the indents are the same inside one YAML file as long as indents are used consistently at the same level inside one logical section - e.g. all items of map/list are are indented the same. 

The controller deployment already featured 14 so I left it that way but of course, can be changed to 12 to align with the rest of the deployments. I can change that, if desired (but it will be one more commit I guess)

Reply:
You can change it to 12 in this commit: )